common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -869,3 +869,17 @@ std::list<uint8_t> wireless_utils::string_to_wsc_oper_class(const std::string &o
         return {};
     }
 }
+
+/**
+ * @brief convert operating class to bandwidth based on Table 4-E in the ieee 802.11 specification
```

Comment:
probably a copy-paste typo? the method brief description is incorrect

Reply:
This is not a related issue.

Reply:
Accepted! Moved Doxygen to the header file, according to the poll results in democracy conversation.